src/utils.js

```diff
@@ -48,7 +48,11 @@ export function mapPathString(nodePath, state) {
     return;
   }
 
-  const modulePath = getRealPath(nodePath.node.value, state);
+  const sourcePath = nodePath.node.value;
+  const currentFile = state.file.opts.filename;
+  const opts = state.opts;
+
+  const modulePath = resolvePath.call(state, sourcePath, currentFile, opts);
```

Comment:
But later (with the new `resolvePath` option) this call will need `opts` as opposed to `normalizedOpts`:
```js
const modulePath = opts.resolvePath(sourcePath, currentFile, opts);
```